packages/grid/devspace.yaml

```diff
@@ -93,19 +101,24 @@ dev:
     labelSelector:
       app.kubernetes.io/name: syft
       app.kubernetes.io/component: backend
-    env:
-      - name: RELEASE
-        value: development
-      - name: DEV_MODE
-        value: "True"
-      - name: DEBUGGER_ENABLED
-        value: "True"
     ports:
```

Comment:
NIT: We would need to expose these ports only in backend-container, maybe we could  add it only to the backend-container-section?

Reply:
@rasswanth-s I tried doing this originally but this is not possible. As per [devspace's docs](https://www.devspace.sh/docs/configuration/dev/selectors/labels#multiple-dev-containers-per-pod)

> since pods share the same network stack, the ports option needs to be on the top-level instead of for each container individually.

Reply:
Right makes sense 👍